### DISCUSSION TITLE
Being able to choose specific query when using Dashboard datasource

### DISCUSSION THREAD

**Discussion by jesjimher** [grafana/grafana#38137] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Right now, when you set Panel's datasource to "Dashboard", in order to reuse other panel's results, there's no way of choosing specific queries from this panel, all of them are imported. It would be nice to select which queries I want to import.

**Why is this needed**:

It's needed because sometimes panels have lots of queries, but you just need one of them.

**Comment by cpmoore** [comment]
+1 for this
I’d also like the ability to specify specific series (preferably by a regex pattern) to include

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
